docs/development/releasing.rst

```diff
@@ -643,7 +690,7 @@ issues that you don't expect to be fixed in time for 'v5.0.2'.
 .. _key-signing-info:
 
 Creating a GPG Signing Key and a Signed Tag
```

Comment:
I, too, think this can be removed altogether, and don't think we need the link as there may be better ones guides coming along, etc, a one sentence should be enough somewhere about the policy whether we need to sign the tags and the release or not. Only technical people will end up doing the release anyway, and for them it's not a far fetched assumption to know how to find and read the relevant git/github manual about signing tags.

Reply:
FWIW, I still think it is a good practice to sign release tags but I agree if there is a more official guide out there, we should not re-invent the doc wheel.

Reply:
I agree that signing them is good, and we should keep doing it, what I don't think is that we need to provide any docs of how to generate the key and how to sign, the instruction: `Sign the tag` and the command example should be enough (instead of the current long writeup).